### ISSUE TITLE
Test Case: Email Composer Sample

### ISSUE THREAD

**Issue by anshuman-mathur** [anshuman-mathur/AppStudio-Testing#6]

**Comment by mahiran** [comment]
I confirmed that this sample works fine.
AppStudio Desktop: Version 3.1.116 (Windows 64 bit) + Cloud Make (Windows 64 bit)
My E-mail app: Microsoft Outlook 2016

**Comment by geoholix** [comment]
There is no Subject title when sending email from apps on laptop (windows os). however subject title is appeared when sending email through mobile device (android)

**Comment by mahiran** [comment]
On my test, probably no problem for Subject title. 
![2018-08-17_10h53_00 composed email](https://user-images.githubusercontent.com/10612837/44243788-e5e30300-a20b-11e8-941f-54121019afef.png)

**Comment by anshuman-mathur** [comment]
@geoholix are you facing this issue with a built app, or while running in from AppStudio or Player?

**Comment by geoholix** [comment]
facing this when running in from AppStudio

**Comment by geoholix** [comment]
I used windows 10 Pro

**Comment by mahiran** [comment]
I'm using Windows 10 Pro 1709)

**Comment by anshuman-mathur** [comment]
@geoholix Ok that is great. This is something we should be investigating and fixing before our release. Could you please create a separate issue as a bug report in this repo. In the bug report can also copy paste the exact code that you have implemented. That might help to narrow down the issue.

**Comment by anshuman-mathur** [comment]
Also have tried to simulate any error as described in steps 6 and 7 yet?

**Comment by mahiran** [comment]
I'll try to test step 6 and 7 on this afternoon.

**Comment by geoholix** [comment]
trying to make wrong picture path in android
![image](https://user-images.githubusercontent.com/1438623/44246950-3312a500-a20b-11e8-8c34-f7eaec9de804.png)

click button - still give response to send email
![image](https://user-images.githubusercontent.com/1438623/44246984-779e4080-a20b-11e8-83c0-634a16be6eb2.png)

email doesn't contain anything
![image](https://user-images.githubusercontent.com/1438623/44247028-c1872680-a20b-11e8-958d-16fa720d8810.png)

close email client, then I got error message
![image](https://user-images.githubusercontent.com/1438623/44247063-f85d3c80-a20b-11e8-9fea-99d5a1ae705b.png)

in my opinion, it should be error message first, then I can continue to send email.

**Comment by mahiran** [comment]
I'm sorry, certainly I don't have enough time for testing step 6 and 7 today.

**Comment by anshuman-mathur** [comment]
@geoholix I agree. The app should show an error message first and then let the user decide if the email should be sent without the attachment or not. Can you please create a bug report describing this issue? We will target to resolve this before the release.

**Comment by anshuman-mathur** [comment]
@mahiran that's ok. Please try to test as much as you comfortably can today. This repo will be accessible to everyone till Monday. Please feel free to provide any feedback in the next few days. Otherwise you can reach us on our GeoNet Forum at https://community.esri.com/groups/appstudio as well.
